.rubocop.yml

```diff
@@ -10,16 +10,16 @@ AllCops:
   DisplayCopNames: true
 
 Layout/LineLength:
-  Max: 120
+  Max: 150
 Metrics/MethodLength:
-  Max: 20
+  Max: 120
 Metrics/AbcSize:
   Max: 50
 Metrics/ClassLength:
   Max: 150
 Metrics/BlockLength:
   IgnoredMethods: ['describe']
-  Max: 30
+  Max: 130
 
```

Comment:
- [ ] @Team it's not good practice to change the default settings of your linter configuration, we are required to use them in default mode to help us write clean and quality code.